tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2020", // default value is ES3
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "allowSyntheticDefaultImports": true,
+    "allowJs": true,
+    "isolatedModules": true,
+    "noImplicitAny": false,
```

Comment:
I think this is the default.

Reply:
I'll double check and remove them if that is the case.

Reply:
you were right, that is the default for `allowSyntheticDefaultImports`. I'm removing it.